src/data/ingredientToIngredientAliases.ts

```diff
@@ -155,6 +155,21 @@ const ingredientToIngredientAliases: IngredientToIngredientAliases = {
   "Boiling Water": [],
   "Cinnamon": [],
   "Pumpkin Syrup": [],
+  "Brandy": [],
+  "Apple Cider": [],
+  "Pear Brandy": [],
+  "Hard Apple Cider": [],
+  "Dry Alsatian Riesling": [],
+  "Star Anise Pod": [],
+  "Ginger": [],
+  "Whole Cloves": [],
+  "Cardamom Pod": [],
+  "Vanilla Bean": [],
+  "Maple": [],
+  "Pear": [],
+  "Allspice": [],
+  "Apple Bitters": [],
```

Comment:
I think (maybe?) searching "Bitters" should show recipes requiring apple bitters 🤔 What do you think?

Reply:
If vanilla and vanilla bean mean the same thing, then yeah.